script.py

```diff
@@ -217,7 +223,20 @@ def parse_commit_content(commit_msg):
 with open(meta_runtime_path, 'w') as f:
     yaml.dump(dataRuntimeMap, f, sort_keys=False, default_flow_style=False, explicit_start=True, Dumper=Dumper)
 
-# 5. Update CHANGELOG.rst & galaxy.yml and push a releasing PR
+# 5. Update annotation unsupported classes
+annotation_unsupported_file = Path('{0}/plugins/module_utils/annotation_unsupported.py'.format(directory))
+apic = os.getenv("APIC")
+if apic and annotation_unsupported_file.exists():
```

Comment:
Is there a reason that the file has to exist should?

Reply:
I think this script runs on multiple repos. I just wanted to make sure it only runs when 'annotation_unsupported.py' has been manually created first.